config/clusters/jupyter-health/cluster.yaml

```
@@ -0,0 +1,28 @@
+name: jupyter-health
+provider: aws
```

cluster.yaml and this comment has been the goto place to figureout where a cluster is available - consistent across all current cluster.yaml files

```suggestion
provider: aws # https://2i2c.awsapps.com/start/#/
```